### PR TITLE
WSTEAMA-1429 Add languages link to footer for all services

### PR DESCRIPTION
**Pull request by LilyL0u**
Resolves JIRA [[1429]](https://jira.dev.bbc.co.uk/browse/WSTEAMA-1429)

Overall changes
======

Added this 
```
{
          href: 'https://www.bbc.com/ws/languages',
          text: 'Other Languages',
        },
```

to the array of link objects in each of the services' config files, after the last one I see present on the current footer.

Code changes
======

- _A bullet point list of key code changes that have been made._

Testing
======

To test locally, check out this branch and visit a page from each service. Check that the 'Other Languages' link is there in the last position in that links section (before copyright at the bottom), and links to  https://www.bbc.com/ws/languages.

Do an a11y check on LTR and RTL that the links are announced correctly by screenreaders as a link that is the last in the list.

Same testing for test and live environments.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/lib/config/services/azeri.ts [hunk 1]
```diff
@@ -302,6 +302,14 @@ export const service: DefaultServiceConfig = {
...
           text: 'BBC ilə Əlaqə',
         },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
```
Comment (amoore108) [inline src/app/lib/config/services/azeri.ts]: Double here

File: src/app/lib/config/services/bengali.ts [hunk 2]
```diff
@@ -311,6 +311,14 @@ export const service: DefaultServiceConfig = {
...
           text: 'বিবিসির সাথে যোগাযোগ করুন',
         },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
```
Comment (amoore108) [inline src/app/lib/config/services/bengali.ts]: Double here

File: src/app/lib/config/services/gujarati.ts [hunk 3]
```diff
@@ -308,6 +308,14 @@ export const service: DefaultServiceConfig = {
...
           text: 'BBC નો સંપર્ક કરો',
         },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
```
Comment (amoore108) [inline src/app/lib/config/services/gujarati.ts]: Double here

File: src/app/lib/config/services/persian.ts [hunk 4]
```diff
@@ -420,6 +420,14 @@ export const service: DefaultServiceConfig = {
...
           text: 'تماس با بی بی سی',
         },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
```
Comment (amoore108) [inline src/app/lib/config/services/persian.ts]: Double here

File: src/app/lib/config/services/somali.ts [hunk 5]
```diff
@@ -314,6 +314,14 @@ export const service: DefaultServiceConfig = {
...
           text: 'La xiriir BBC',
         },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
```
Comment (amoore108) [inline src/app/lib/config/services/somali.ts]: Double here

**Review by amoore108 — COMMENTED** [review]
Couple of dubs!

File: src/app/lib/config/services/azeri.ts (hunk 1 above)
Comment (LilyL0u) [reply src/app/lib/config/services/azeri.ts]: So good I linked it twice. Will remove these overexcited copy pastes.

**Review by amoore108 — APPROVED** [review]

**Review by karinathomasbbc — APPROVED** [review]

**Pull request merged**